lib/net.js

```diff
@@ -133,6 +134,7 @@ function Socket(options) {
   this._handle = null;
   this._parent = null;
   this._host = null;
+  this._timeout = null;
```

Comment:
Not that I'm able to see. There are zero hits anywhere in the source for a `_timeout` property